src/nvme/ioctl.h

```diff
@@ -3880,7 +3886,9 @@ struct nvme_io_args {
 	__u16 appmask;
 	__u16 dspec;
 	__u8 dsm;
-};
+	__u8 sts;
+	__u8 pif;
+} __attribute__((__packed__, aligned(__alignof__(__u64))));
```

Comment:
Again no need for these attributes.

Reply:
I had to keep the packed attribute to get the exact structure size.  Without it the structure is padded out to a 4 byte boundary and the size is not different between the old and new versions of the struct.

Reply:
Oh, you are right. I hoped we don't need those attributes...